eth/filters/filter.go

```diff
@@ -142,6 +161,9 @@ func (f *Filter) Logs(ctx context.Context) ([]*types.Log, error) {
 	if f.end == -1 {
 		end = head
 	}
+	if end-uint64(f.begin) > blockRangeLimit {
+		return nil, errors.New("filter block range is higher than the limit")
```

Comment:
@minh-bq  Could u print `end-uint64(f.begin)` as well for tracking

Reply:
Added